src/automaticassessmentconfiguration.cs

```diff
@@ -118,5 +142,41 @@ interface AEAssessmentSessionDelegate {
 
 		[Export ("assessmentSessionDidEnd:")]
 		void DidEnd (AEAssessmentSession session);
+
+		[NoiOS, Mac (12,0), MacCatalyst (15,0)]
+		[Export ("assessmentSessionDidUpdate:")]
+		void DidUpdate (AEAssessmentSession session);
```

Comment:
No iOS but MacCatalyst? That is weird.

Reply:
Oh yeah.. that is interesting

Reply:
Looking at the docs, is is correct, and that is why I approved, it is just funny that Catalyst was meant to be iOS apps that run on Mac, yet they do things like this. 🍎